box-share-sdk/src/main/java/com/box/androidsdk/share/vm/InviteCollaboratorsDataWrapper.java

```diff
@@ -0,0 +1,40 @@
+package com.box.androidsdk.share.vm;
+
+/**
+ * A special extension of DataWrapper to hold data for inviting a new Collaborator
+ */
+public class InviteCollaboratorsDataWrapper extends DataWrapper<String> {
+
+    private boolean mInvitationFailed;
+
+    public InviteCollaboratorsDataWrapper(String data, int strCode, boolean invitationFailed) {
+        this.mData = data;
+        this.mStrRes = strCode;
+        this.mInvitationFailed = invitationFailed;
+    }
+
+    /**
+     * Returns true if all invitations succeeded.
+     * @return true if all invitations succeeded
+     */
+    @Override
+    public boolean isSuccess() {
+        return !isInvitationFailed();
+    }
+
+    /**
+     * Returns true if at least one invitation failed.
+     * @return true if at least one invitation failed
+     */
+    public boolean isInvitationFailed() {
+        return mInvitationFailed;
+    }
+
+    /**
+     * Returns true if the String formatted part exists.
```

Comment:
Not sure what this means

Reply:
Some string resource files are used together with String.format. This is to signify there exists a data that should be passed in to String.format. 

Ideally I should be retrieving the string from the string resource and checking. But currently, if the developer pass this data in I am assuming he would pass in the string resource that has %s in it to allow data to be passed in.